CleanWeather/Networking/APIUrls.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  APIUrls.swift
+//  CleanWeather
+//
+//  Created by Marek Skrzelowski on 12/12/2019.
+//  Copyright © 2019 vandermesis. All rights reserved.
+//
+
+import Foundation
+
+struct APIUrls {
+
+    static let opendata = "public.opendatasoft.com"
```

Comment:
tu ma być cała ścieżka bez path, razem z https

Reply:
Jak dawałem razem z https:// to potem podczas requestu robiły mi się krzaczki.
`//http%3A%2F%2Fpublic.opendatasoft.com/api/records/1.0/search/?dataset=worldcitiespop&lang=pl&sort=population&facet=coun ... untry=pl`
Na razie dodałem scheme(defatult https) do Requestu i w taki sposób działa.